src/languages/python.ts

```diff
@@ -79,7 +82,17 @@ const nodeMatchers: Partial<Record<ScopeType, NodeMatcherAlternative>> = {
     ),
     patternMatcher("return_statement.~return!")
   ),
-  argumentOrParameter: argumentMatcher("parameters", "argument_list"),
+  argumentOrParameter: cascadingMatcher(
+    argumentMatcher("parameters", "argument_list"),
+    matcher(
+      ancestorChainNodeFinder(
+        1,
+        patternFinder("call"),
```

Comment:
Heyo! I think you could do this using the language matching syntax: 

```
matcher(
      patternFinder("call.generator_expression!"),
      childRangeSelector()
    )
```   
What is happening here is — selecting node if the parent is a `call` type and the child is a `generator_expression` type. Without the `"!"`, we would select all of these nodes, however, we just want to select the arguments, not the entire call.

Reply:
I've made this change and verified it still works.